src/java/org/apache/cassandra/index/sai/plan/StorageAttachedIndexSearcher.java

```diff
@@ -127,6 +127,7 @@ public UnfilteredPartitionIterator search(ReadExecutionController executionContr
         // Can't check for `command.isTopK()` because the planner could optimize sorting out
         if (plan.ordering() != null)
         {
+            queryContext.setOptimizedPlan(plan.toStringRecursive());
```

Comment:
If this String may contain user data then we have to redact it otherwise we must ask to the customer the permission to use this tool  that means that we will use it only in tedt and dev end and it would be a pity

Reply:
Well, my understanding was at first we want the slowest queries in test/dev to easy identify and debug them while stress testing.

I will look into the redacting, thanks.